backend-go/internal/server/server.go

```
@@ -3,6 +3,7 @@ package server
 import (
 	"errors"
 	"fmt"
+	"github.com/yorkie-team/codepair/backend/internal/middleware"
```

golangci-lint (1.62.2)

File is not `goimports`-ed with -local github.com/yorkie-team/codepair

(goimports)